src/spec/expectations.cr

```diff
@@ -1,4 +1,58 @@
 module Spec
+  # A list of diff command candidates.
+  DIFF_COMMANDS = %w(gdiff diff)
```

Comment:
These commands will almost never be available on Windows.

`git diff --no-index -U3` is another option for this. Funnily enough, on Windows that one is way more likely to be present.